app/views/videos/show.html.erb

```diff
@@ -17,28 +17,27 @@
 <div class="container shop">
   <p class="title" style="font-size: 18px; text-align: center;">Shop the Look</p>
   <br>
-  <% @products.each do |prod| %>
+  <% @annotations.each do |a| %>
+    <p id="p_<%= "#{a.product.id}" %>"></p>
     <div class="card-video card-prod">
       <img src="https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/skateboard.jpg" />
-    <div class="card-video-infos">
-      <h2><%= link_to prod.name, prod %></h2>
-      <p><%= prod.description %></p>
-    </div>
-    <div class="card-video-infos info-right">
-      <h2>$<%= prod.price %></h2>
+      <div class="card-video-infos">
+        <h2><%= link_to a.product.name, a.product %></h2>
+        <p><%= a.product.description %></p>
+      </div>
+      <div class="card-video-infos info-right">
+        <h2 >$<%= a.product.price_cents %></h2>
```

Comment:
You can use <%= humanized_money_with_symbol a.product.price_cents %>
It is slightly nicer